src/components/engagement/dto/engagement.dto.ts

```diff
@@ -55,6 +56,12 @@ export const resolveEngagementType = (val: Pick<AnyEngagement, '__typename'>) =>
     ? LanguageEngagement
     : InternshipEngagement;
 
+const RequiredWhenNotInDev = RequiredWhen(() => Engagement)({
+  description: 'the engagement is not in development',
+  isEnabled: ({ status }) =>
+    status !== 'InDevelopment' && status !== 'DidNotDevelop',
+});
```

Comment:
I guess there's no way to DRY this with the one in the Project dto?

Reply:
The description is different. Beyond that we could but it's not worth it IMO. The fact that the actual check happens to be the same is anecdotal, they are different enums.